client/package.json

```diff
@@ -6,11 +6,12 @@
     "@material-ui/core": "^4.1.3",
     "@material-ui/icons": "^4.9.1",
     "create-react-app": "^3.4.1",
+    "json2mq": "*",
```

Comment:
I removed this package because it was not needed in frontend, not seeing it used anywhere in the repo, wondering if it's being added back intentionally

Reply:
I don't remember. It's been a while. I'll check if it's being used anywhere

Reply:
I don't think that's needed.